### PR TITLE
Picture Layout Darkmode

### PR DESCRIPTION
**Pull request by DanielCliftonGuardian**
## What does this change?
Implements dark mode for picture layout
## Why?
Resolves https://github.com/guardian/dotcom-rendering/issues/9586
## Screenshots

| Light      | Dark      |
| ----------- | ---------- |
| ![before][] | ![after][] |

[before]: https://github.com/guardian/dotcom-rendering/assets/110032454/8250ec0f-ed34-4753-9edb-c9bb541f7aac
[after]: https://github.com/guardian/dotcom-rendering/assets/110032454/81ef8566-b94c-4195-99f4-0869813e28b5

### CODE REVIEW

File: dotcom-rendering/src/layouts/PictureLayout.tsx [hunk 1]
```diff
@@ -408,17 +412,17 @@ export const PictureLayout = (props: WebProps | AppsProps) => {
...
 					</Island>
 				)}
 				<Section
 					fullWidth={true}
 					showTopBorder={false}
-					backgroundColour={palette.background.article}
+					backgroundColour={themePalette('--article-background')}
 					element="article"
-					borderColour={palette.border.secondary}
+					borderColour={themePalette('--article-border')}
```
Comment (abeddow91) [inline dotcom-rendering/src/layouts/PictureLayout.tsx:425]: I think this should be 
```suggestion
					borderColour={themePalette('--article-border-secondary')}
```

File: dotcom-rendering/src/layouts/PictureLayout.tsx [hunk 2]
```diff
@@ -563,7 +571,7 @@ export const PictureLayout = (props: WebProps | AppsProps) => {
 						<GridItem area="submeta">
 							<ArticleContainer format={format}>
 								<DecideLines
-									color={palette.border.secondary}
+									color={themePalette('--article-border')}
```
Comment (abeddow91) [inline dotcom-rendering/src/layouts/PictureLayout.tsx:574]:
```suggestion
									color={themePalette(
												'--article-border-secondary'
											)}
```

**Review by abeddow91 — CHANGES_REQUESTED** [review]

File: dotcom-rendering/src/layouts/PictureLayout.tsx [hunk 3]
```diff
@@ -251,6 +252,9 @@ export const PictureLayout = (props: WebProps | AppsProps) => {
 		config: { isPaidContent, host },
 	} = article;
 
+	const isWeb = renderingTarget === 'Web';
+	const isApps = renderingTarget === 'Apps';
```
[the remark this answers is not in the document]
Comment (mxdvl) [reply dotcom-rendering/src/layouts/PictureLayout.tsx:256]: I’m not sure this is more readable, actually, now there’s one more layer of abstraction.

**Review by abeddow91 — APPROVED** [review]

File: dotcom-rendering/src/layouts/PictureLayout.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (mxdvl) [reply dotcom-rendering/src/layouts/PictureLayout.tsx:256]: No strong feelings either, but two against one you win 🎉

**Pull request merged**